src/VisualStudio/Core/Def/Remote/VisualStudioRemoteHostClientProvider.cs

```diff
@@ -56,18 +61,27 @@ public IWorkspaceService CreateService(HostWorkspaceServices workspaceServices)
                 // We don't want to bring up the OOP process in a VS cloud environment client instance
                 // Avoids proffering brokered services on the client instance.
                 if (!_globalOptions.GetOption(RemoteHostOptionsStorage.OOP64Bit) ||
-                    workspaceServices.Workspace is not VisualStudioWorkspace ||
+                    workspaceServices.Workspace is not (VisualStudioWorkspace or PreviewWorkspace) ||
```

Comment:
this turned out to be much easier than i thought it would be.  Thanks to Sam for helping me understand mef here.